templates/index.html

```diff
@@ -277,7 +277,47 @@ <h2>Experience MicroK8s</h2>
    </div>
  </section>
 
- <section class="p-strip is-deep">
+  <section class="p-strip">
+    <div class="row">
+      <h2>MicroK8s and Kubernetes resources</h2>
+    </div>
+    <div class="row p-divider">
+      <div class="col-4 p-divider__block">
+        <div class="p-heading-icon--small">
+          <div class="p-heading-icon__header">
+            <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/5edefef9-Datasheet.svg" alt="">
+            <h4 class="p-heading-icon__title">Webinars</h4>
```

Comment:
you can add `p-heading--4` to make sure the [heading hierarchy](https://www.w3.org/WAI/tutorials/page-structure/headings/) remains correct
```suggestion
            <h3 class="p-heading-icon__title p-heading--4">Webinars</h4>
```